### PR TITLE
User committee test

### PR DESCRIPTION
**Pull request by MarcSaureu**
User - CommitteeMember relationship Test Done

### CODE REVIEW

File: src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java [hunk 1]
```diff
@@ -52,6 +56,9 @@ public void itHasBeenAssignedTheRankToAUser(String rank) throws Exception {
     public void iAssignARankToAUserWithUsername(String rank, String username) throws Exception {
         CommitteeMember committeeMember = new CommitteeMember();
         committeeMember.setRank(rank);
+        User user = userRepository.findByUsernameContaining(username).get(0);
+        System.out.println(user);
```
Comment (rogargon) [inline src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java:60]: Please, remove println

File: src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java [hunk 2]
```diff
@@ -61,4 +68,20 @@ public void iAssignARankToAUserWithUsername(String rank, String username) throws
                         .with(AuthenticationStepDefs.authenticate()))
                 .andDo(print());
```
Comment (rogargon) [inline src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java:69]: You already have the .andDo(print()) to debug your code so no need for the println

File: src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java [hunk 3]
```diff
@@ -61,4 +68,20 @@ public void iAssignARankToAUserWithUsername(String rank, String username) throws
...
     }
+
+    @And("It has been assigned the rank {string} to a user with username {string}")
+    public void itHasBeenAssignedTheRankToAUserWithUsername(String rank, String username) throws Exception {
+        stepDefs.result = stepDefs.mockMvc.perform(
+                get("/committeeMembers/{id}", 1L)
```
Comment (rogargon) [inline src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java:75]: The committeeMember identifier is here fixed to 1. Better to keep dynamic, get the id using findBy or storing the URI of the new resource retrieving the Location header from the POST response when it is created

**Review by rogargon — CHANGES_REQUESTED** [review]

**Review by MarcSaureu — COMMENTED** [review]
Deleted Println and Dynamic Id

**Review by rogargon — APPROVED** [review]

**Pull request merged**